server/app/services/UserService.ts

```diff
@@ -1,15 +1,26 @@
 import { IUserEntity, UserEntity } from './../entities/User';
-import { IUser } from './../models/UserModel';
+import { IUser, IEmail } from './../models/UserModel';
 import { UserRepository } from './../repository/UserRepository';
 import { inject, injectable } from 'inversify';
 import TYPES from '../constants/types';
 import bcrypt from 'bcryptjs';
-import { IConfig } from 'config';
+import config, { IConfig } from 'config';
+import nodemailer, { SentMessageInfo } from 'nodemailer';
+import Mail from 'nodemailer/lib/mailer';
 
 @injectable()
 export class UserService {
+  private transporter: Mail;
   @inject(TYPES.config) private config: IConfig;
-  constructor(@inject(TYPES.UserRepository) private userRepo: UserRepository) {}
+  constructor(@inject(TYPES.UserRepository) private userRepo: UserRepository) {
+    this.transporter = nodemailer.createTransport({
```

Comment:
I would add this nodemailer.createTransport call to the registery.ts file so you can just inject constant value like we do for a logger

Reply:
example: https://github.com/Mackbellemore/soen-390-team07/blob/develop/server/app/registry.ts#L27

Reply:
great! I will make the changes

Reply:
Also I would create a EmailService.ts to have this seperately along with a EmailController.ts file